database/1-create-database.sql

```diff
@@ -0,0 +1,34 @@
+
+
+DROP TABLE guest;
+DROP TABLE booking;
+
+use hotel;
+
+CREATE TABLE booking (
+
+	id INT AUTO_INCREMENT,
+	entry_date DATE,
+	departure_date DATE,
+	price DOUBLE(50),
+	method_payment VARCHAR(20),
+	PRIMARY KEY(id)
+)Engine=InnoDB;
+
+CREATE TABLE guest (
+
+	id INT AUTO_INCREMENT,
+	name VARCHAR(50)NOT NULL,
+	lastname VARCHAR(100)NOT NULL,
+	birthdate DATE,
+	nationality VARCHAR(50),
+	(telephone VARCHAR(15), CONSTRAINT chk_telephone CHECK (telephone REGEXP '^[0-9]{10}$'))
```

Comment:
Please update the regular expression to be like this:
```
^[0-9]{7,10}$
```
This will allow numbers from 7 to 10 digits, like before it allowed only 10 number digits.